1.0.14/init_container.sh

```diff
@@ -20,16 +20,27 @@ service ssh start
 
 [ -z "$ASPNETCORE_URLS" ] && export ASPNETCORE_URLS=http://*:"$PORT"
 
-# If there is any command line argument specified, run it
+# If there is any command line argument specified, do basic validation and execute.
 if [ $# -ne 0 ]; then
-   echo "App Command Line configured: $@"
-   echo "Checking if first argument is dotnet and second argument is a file on disk"
-   if [ $1 == "dotnet" -a -f $2 ]; then
-        echo "Argument to dotnet exists on disk, launching it:  $@"
+    echo "App Command Line configured: $@"
+    if [ $# -eq 1 ]; then
+        # ensure $1 exists on disk
+        if [ -f $1 ]; then
+            echo "App Command Line file on disk, executing it."
+            exec "$@"
+        else
+            echo "App command Line file doesn't exist on disk, fallback to auto-detect"
+        fi
+    elif [ $# -eq 2 -a $1 == "dotnet" ]; then
+        if [ -f $2 ]; then
+            echo "Argument to dotnet exists on disk, launching it: $@"
+       	    exec "$@"
+        else
+            echo "Argument to dotnet doesn't exist on disk, fallback to auto-detect"
+        fi
+    else
         exec "$@"
-   else
-        echo "App command line ignored, since file $2 was not found"
-   fi
+    fi
 fi
```

Comment:
I think this should be an else, as we shouldnt be executing the rest if we find a app command line